src/Microsoft.Identity.Web.TokenAcquisition/DefaultAuthorizationHeaderProvider.cs

```diff
@@ -49,19 +49,9 @@ public async Task<string> CreateAuthorizationHeaderForAppAsync(string scopes, Au
 
         private static TokenAcquisitionOptions CreateTokenAcquisitionOptionsFromApiOptions(AuthorizationHeaderProviderOptions? downstreamApiOptions, CancellationToken cancellationToken)
         {
-            return new TokenAcquisitionOptions()
-            {
-                AuthenticationOptionsName = downstreamApiOptions?.AcquireTokenOptions.AuthenticationOptionsName,
-                CancellationToken = cancellationToken,
-                Claims = downstreamApiOptions?.AcquireTokenOptions.Claims,
-                CorrelationId = downstreamApiOptions?.AcquireTokenOptions.CorrelationId ?? Guid.Empty,
-                ExtraQueryParameters = downstreamApiOptions?.AcquireTokenOptions.ExtraQueryParameters,
-                ForceRefresh = downstreamApiOptions?.AcquireTokenOptions.ForceRefresh ?? false,
-                LongRunningWebApiSessionKey = downstreamApiOptions?.AcquireTokenOptions.LongRunningWebApiSessionKey,
-                Tenant = downstreamApiOptions?.AcquireTokenOptions.Tenant,
-                UserFlow = downstreamApiOptions?.AcquireTokenOptions.UserFlow,
-                PopPublicKey = downstreamApiOptions?.AcquireTokenOptions.PopPublicKey,
-            };
+            TokenAcquisitionOptions tokenAcquisitionOptions = TokenAcquisitionOptions.CloneFromBaseClass(downstreamApiOptions?.AcquireTokenOptions);
+            tokenAcquisitionOptions.CancellationToken = cancellationToken;
+            return tokenAcquisitionOptions;
```

Comment:
Why not just add 
`ManagedIdentity = downstreamApiOptions?.AcquireTokenOptions.ManagedIdentity`?

Reply:
I don't understand why you are introducing a new method here, @JoshLozensky?
As @jennyf19 asked, wasn't adding this line enough?

Reply:
This still needs to be resolved before we can merge.

Reply:
I was trying to uncouple the logic from the DefaultAuthorizationHeaderProvider and move it into the child class where it will be easier to find in the future to me it was really unintuitive that if I make changes to TokenAcquisitionOptions in abstractions I needed to also edit this ostensibly unrelated class in order for it to work. If it is in the child class it keeps the logic with the object it belongs to.

This isn't a hill to die on for me so if you'd prefer I'll just put it back @jennyf19 and @jmprieur.